RI.md

```diff
@@ -0,0 +1,842 @@
+#`ri`: Ruby Information
+
+`ri` (<b>r</b>uby <b>i</b>nformation) is the Ruby command-line utility
+that gives fast and easy on-line access to Ruby documentation.
+
+`ri` can show documentation for Ruby itself and for its installed gems:
+
+- A **class** or **module**:
+  text associated with the class or module definition
+  in a source file (`.rb` or `.c`).
+- One or more **methods**:
+  text associated with method definitions
+  in source files (`.rb` and `.c`).
+- A **page**:
+  text from a stand-alone documentation file
+  (`.rdoc` or  `.md`, or sometimes other).
+
+Examples (output omitted):
+
+```sh
+$ ri Hash             # Document for class Hash.
+$ ri Array#sort       # Document for instance method sort in class Array.
+$ ri read             # Documents for methods ::read and #read in all classes and modules.
+$ ri ruby:dig_methods # Document for page dig_methods.
+```
+
+`ri` can also show lists of:
+
+- **classes** and **modules**:
+  full or partial list.
+- **pages**:
+  for Ruby or for an installed gem.
+
+Examples (output omitted):
+
+```sh
+$ ri --list # List of classes and modules.
+$ ri ruby:  # List of Ruby pages.
+```
+
+## Why `ri`?
+
+Using `ri` may have advantages over using
+the [Ruby online documentation](https://docs.ruby-lang.org/en/master):
+
+- The `ri` documentation is always available, even when you do not have internet access
+  (think: airplane mode).
+- If you are working in a terminal window, typing `ri _whatever_` (or just `ri`)
+  may be faster than navigating to a browser window and searching for documentation.
+- If you are working in an
+  [irb \(interactive Ruby\)](https://docs.ruby-lang.org/en/master/IRB.html)
+  session, you _already_ have immediate access to `ri`:
+  just type `'help'` or `'show_doc'`.
```

Comment:
In the latest IRB we've repurposed `help` to show the help message of IRB itself 🙂 

```suggestion
  just type `'show_doc'`.
```

Reply:
Fixed.